src/widgets/medications/medication-order-basket.component.tsx

```diff
@@ -224,27 +224,23 @@ export function MedicationOrderBasket(props: MedicationOrderBasketProps) {
         </div>
       </div>
 
-      <div style={{ width: "70%" }}>
+      <div style={{ width: "90%" }}>
         {orderBasket.length > 0 &&
           orderBasket.map((order, index) => {
             return (
               <div
-                className={`${styles.basketStyles} ${
-                  order.action === NEW_MEDICATION_ACTION
-                    ? styles.newOrder
-                    : styles.reviseOrder
-                }`}
+                className={`${styles.basketStyles} ${styles.OrderStyle}`}
                 key={index}
               >
                 <SummaryCardRow>
                   <SummaryCardRowContent justifyContent="space-between">
                     <span>
-                      <b>{order.drugName}</b>
+                      {order.action} <b>{order.drugName}</b>
                       {" \u2014 "}{" "}
                       {String(order.dosageForm).toLocaleLowerCase()}
                       {" \u2014 "} {String(order.routeName).toLocaleLowerCase()}
-                      {" \u2014 "}{" "}
-                      {`DOSE ${getDosage(order.drugStrength, order.dose)}`}{" "}
+                      {" \u2014 "} DOSE{" "}
```

Comment:
write a helper function to help you concatenate this, it would make it cleaner to read.